src/components/ColorPicker/ColorPicker.vue

```diff
@@ -32,10 +32,55 @@ actual pickers:
 
 ### Usage
 
+* Using v-model and passing in an HTML element that will be treated as a trigger:
+
+```vue
+<template>
+	<ColorPicker v-model="color">
+		<button> Click Me </button>
```

Comment:
Why is this button indented and what does it do in the first place? It doesn't have any `@click` whatsoever

Reply:
It's passed in into a slot and treated as a trigger by the v-popover component. It can actually be any HTML element, not necessarily a button.

Reply:
But in that case the next line should be `</ColorPicker>`, not `<ColorPicker />` :)

Reply:
Indeed :)